java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -739,4 +739,46 @@ public ResponseEntity<Object> getAttestationCertificate(HttpServletRequest reque
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
     }
+    @RequestMapping(value = "/api/v1/{entityName}/{entityId}/revoke", method = RequestMethod.POST)
+    public ResponseEntity<Object> revokeACredential (
+            HttpServletRequest request,
+            @PathVariable String entityName,
+            @PathVariable String entityId,
+            @RequestHeader HttpHeaders headers
+    ){
+        String userId = USER_ANONYMOUS;
+        logger.info("Revoking the entityType {} with {} Id",entityName, entityId);
+        // Check fot Authorisation
+        if (registryHelper.doesEntityOperationRequireAuthorization(entityName)) {
+            try {
+                userId = registryHelper.authorize(entityName, entityId, request);
+            } catch (Exception e) {
+                return createUnauthorizedExceptionResponse(e);
+            }
+        }
+        ResponseParams responseParams = new ResponseParams();
+        Response response = new Response(Response.API_ID.REVOKE, "OK", responseParams);
+        try {
+            String tag = "RegistryController.revokeAnExistingCredential" + entityName;
```

Comment:
add space after revokeAnExistingCredential

Reply:
Have added the space>